File: leet code practice/Minimum Number of Operations to Move All Balls to Each Box.py
class Solution:
    def minOperations(self, boxes: str) -> List[int]:
        arr=[]
        res=[]
        for i in range(len(boxes)):
            if boxes[i] == "1" :
                arr.append(i)
    
        for i in range(len(boxes)):
            sum=0
            for j in arr :
                sum+=abs(j-i)
            res.append(sum)
        return res
            

File: leet code practice/Minimum Number of Operations to Move All Balls to Each Box.js
var minOperations = function(boxes) {
    var map=[]
    var res=[]
    for (let i=0;i<boxes.length;i++){
        if(boxes[i]==1)
            map.push(i)
    }
    for(let i=0;i<boxes.length;i++){
        var sum=0
        for(j of map){
            sum+=Math.abs(j-i)
        }
        res.push(sum)
    }
    return res
}